.github/actions/deploy-happy-stack/action.yml

```diff
@@ -38,18 +42,25 @@ runs:
         CREATE_TAG: ${{ inputs.create-tag }}
         TAG: ${{ inputs.tag }}
         ENV: ${{ inputs.env }}
+        OPERATION: ${{ inputs.operation }}
       shell: bash
       run: |
         set -ue
         set -o pipefail
         
         IMAGE_TAG="${TAG:-sha-$GITHUB_SHA}"
         echo "Targetting Stack ${STACK_NAME}, tagging with '${IMAGE_TAG}'"
+        HAPPY_OPERATION=${OPERATION}
+        HAPPY_DEPLOYMENT_FLAGS=""
 
-        if $(happy list --aws-profile "" --env=${ENV} 2>&1 | grep -q ${STACK_NAME} ); then
-          echo "Updating stack ${STACK_NAME}"
-          happy update --aws-profile "" --env=${ENV} --create-tag=${CREATE_TAG} --tag ${IMAGE_TAG} ${STACK_NAME}
-        else
-          echo "Creating stack ${STACK_NAME}"
-          happy create --aws-profile "" --env=${ENV} --create-tag=${CREATE_TAG} --tag ${IMAGE_TAG} ${STACK_NAME}
+        if [[ ${HAPPY_OPERATION} != "create" && ${HAPPY_OPERATION} != "update" && ${HAPPY_OPERATION} != "delete" ]]; then
```

Comment:
should just check for emptiness here, if it is set then we should respect the operation given